src/lib/y2storage/encryption_method.rb

```diff
@@ -60,14 +62,24 @@ def initialize(id, label, process_class)
     LUKS1 = new(
       :luks1, N_("Regular LUKS1"), EncryptionProcesses::Luks1
     )
+
     PERVASIVE_LUKS2 = new(
       :pervasive_luks2, N_("Pervasive Volume Encryption"), EncryptionProcesses::Pervasive
     )
+
     RANDOM_SWAP = new(
-      :random_swap, N_("Random Swap"), EncryptionProcesses::Swap
+      :random_swap, N_("Random Swap"), EncryptionProcesses::RandomSwap
+    )
+
+    PROTECTED_SWAP = new(
+      :protected_swap, N_("Protected Swap"), EncryptionProcesses::ProtectedSwap
     )
 
-    ALL = [LUKS1, PERVASIVE_LUKS2, RANDOM_SWAP].freeze
+    SECURE_SWAP = new(
+      :secure_swap, N_("Secure Swap"), EncryptionProcesses::SecureSwap
```

Comment:
Same than before. I think "Random Swap with Secure Key" describes this much better. Or maybe something like "Volatile Encryption with Random/Protected/Secure Key"